lua/gruvbox/colors.lua

```diff
@@ -31,10 +31,12 @@ function M.setup(config)
     cyan = "#89b482",
     purple = "#d3869b",
     orange = "#e78a4e",
+    orange2 = "#c35e0a";
     yellow = "#d8a657",
     yellow2 = "#b47109",
     bg_yellow = "#a96b2c",
     green = "#a9b665",
+    aqua = "#8ec07c",
```

Comment:
By the way cyan is the equivalent of aqua. unless we refactor cyan to aqua I don't see a reason to add this color

Reply:
After answering this questing and dealing with the aqua/cyan issue I am willing to merge it. I think I looks slightly better with your changes. 
Let me know :)

Reply:
I removed aqua and replaced it where applicable with cyan.  I'd be happy to refactor cyan to aqua variable naming if you prefer, but completely up to you.  I have no preference either way.

Really appreciate your help working through this :)

Let me know if I can do anything else

Reply:
To be consistent with other Gruvbox themes I think it'll be better if we name it aqua. So go for it, this might seem like it's dragging I should've done a better code review initially. Sorry for the inconveniences

Reply:
Not a problem!  I'll get this taken care of :)